uwblueprint/sistema
# Calendar functionality

# Notion ticket link



[Calendar Absence Creation and Deletion](https://www.notion.so/uwblueprintexecs/Calendar-Absence-Creation-and-Deletion-63a90cae5ec2442ebe1f7aed5ab6bd87?pvs=4)



## Implementation description

- Implemented a fully functional calendar with the ability to add and delete absences which gets stored in the database and is in sync 



## Steps to test

run this in light mode as it's not styled yet completely for dark mode

1. Ensure the database is running
2. `npm run dev` and go to `localhost:3000/calendar`
3. Try adding and deleting multiple absences and check if they are getting updated in the database 




## What should reviewers focus on?

- Absences getting synced with the database and the calendar on the frontend

## Checklist
